v2/bootstrap/container/common.go

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
+	v2Clients "github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+)
+
+// CommonClientName contains the name of the CommonClient instance in the DIC.
+var CommonClientName = "V2CommonClient"
```

Comment:
Please use this form of creating the name for all these containers. 
 di.TypeInstanceToName((*interfaces.CommonClient)(nil))
This will create the name with the full path containing `v2` and will not need to changed once we drop `v2` when v1 code is removed.

Reply:
Thanks!  Modified through https://github.com/edgexfoundry/go-mod-bootstrap/pull/158/commits/d642159ca0d71c54bb031435381dd86a003c3403